### DISCUSSION TITLE
如何关闭单个应用的内置禁用

### DISCUSSION THREAD

**Discussion by ahiswell** [gkd-kit/gkd#817] [Q&A] [answered]
有个应用开启不了全局的全屏广告，显示内置禁用
![619c68451b038842651f4ec5ef92e15](https://github.com/user-attachments/assets/0a13598c-8b3f-4d1e-895f-e9707e1fdd1d)

**Comment by lisonge** [comment]
看不懂描述吗，弹窗内文本已经说得很清楚了

**Reply by ahiswell** [reply]
怎么说呢，我意思是有没有什么功能能解除这个限制，也用上全局的规则，因为实测这个app的广告是可以跳过的

**Reply by lisonge** [reply, marked as answer]
用户解除不了，去找订阅维护者叫他开一下，或者自己把规则复制出来自己改把禁用去掉

**Reply by ahiswell** [reply]
gkd很好用，希望后续可以从软件上增加解除功能
